components/SectionRoadmap.vue

```diff
@@ -105,15 +107,15 @@ export default {
       },
     }
   },
-  computed: {
-    progressTotal() {
-      const progressSum = this.milestoneList
-        .map((i) => Math.floor(i.progress))
-        .reduce((a, b) => a + b, 0)
-      const percentage = ((progressSum / 300) * 100).toFixed(0)
-      return percentage
-    },
-  },
+  // computed: {
```

Comment:
Is there a reason for keeping commented code?

Reply:
If we need it later down the road. I can see this being reused in the future. It's also part of the reason why I kept the old components (SectionNavOld, SectionTestnetsOld).